.circleci/config.yml

```diff
@@ -1,6 +1,16 @@
 version: 2
 
 jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.11.4
+    working_directory: /go/src/github.com/cloudwan/gohan
+    environment:
+        GO111MODULE: 'on'
+    steps:
+      - checkout
+      - run: go version
+      - run: make deps gen lint build
```

Comment:
As per @cierpuchaw 's comment, we can simply test lint and build here. I said that it's ok to remove, but let's revert this job .